third_party/mbedtls/Makefile.am

```diff
@@ -62,6 +62,7 @@ libmbedtls_la_SOURCES                   = \
 libmbedtls_la_CPPFLAGS                        = \
     -I$(srcdir)/repo/include                    \
     -I$(srcdir)/repo/configs                    \
+    -D_GNU_SOURCE                               \
```

Comment:
I didn't override warning flags for mbedtls.

`third_party/mbedtls/repo/library/entropy_poll.c:105` uses `syscall()`, which from [man page](http://man7.org/linux/man-pages/man2/syscall.2.html), requires this flag.